### ISSUE TITLE
Scrollbar shrinks when mouse pointer is on the edge of the screen (scrollbar control padding area)

### ISSUE THREAD

**Issue by hakimio** [MurhafSousli/ngx-scrollbar#300]
#### Reproduction

Steps to reproduce:
1. Position the scrollbar at the edge of the screen
2. Set scrollbar size and hover sizes to different values
3. Move your mouse to the very edge of the screen
 
 
#### Expected Behavior

The hover size should be used when mouse pointer is at the very edge of the screen.

#### Actual Behavior

When the mouse pointer is at the very edge, scrollbar contracts again.

Long story short, the hover size should be applied when hovering over `scrollbar-y.scrollbar-control` element, not on child `.ng-scrollbar-track`.

**Comment by MurhafSousli** [comment]
Please provide a screenshot or video reproduction if possible

**Comment by MurhafSousli** [comment]
I don't think this is the behavior everyone wants, some people only want the hover size to appear when the pointer is over the actually scrollbar not including the padding

**Comment by hakimio** [comment]
Current behavior causes flickering when user hovers on the tiny "padding". I am sure nobody wants that.
Also, "perfect scrollbar" doesn't have this "feature".

**Comment by MurhafSousli** [comment]
Why wouldn't you remove the tiny padding?

**Comment by hakimio** [comment]
Setting `--scrollbar-padding` to 0 causes ngx-scrollbar to break.

**Comment by hakimio** [comment]
Also, it looks pretty bad without the padding.

**Comment by MurhafSousli** [comment]
Use `0px` not 0 because the css variable is involved in other CSS calculation

**Comment by hakimio** [comment]
Still my point stands: it looks bad without the padding and flickering of the thumb width is unwanted behavior.

**Comment by MurhafSousli** [comment]
Let's keep this open and see if other users is also interested in this

**Comment by hakimio** [comment]
If you don't agree and don't plan to do anything about it anytime soon, feel free to just close with "works as intended".

**Comment by MurhafSousli** [comment]
merged in v7.4.0
